consumers/src/main/java/uk/gov/moj/sdt/consumers/IndividualRequestConsumer.java

```diff
@@ -203,22 +200,12 @@ private IndividualResponseType invokeTargetAppService(final IndividualRequestTyp
     /**
      * @param transformer the transformer
      */
-    // CHECKSTYLE:OFF
     public void
-    setTransformer(final IConsumerTransformer<IndividualResponseType, IndividualRequestType, IIndividualRequest, IIndividualRequest> transformer) {
+    setTransformer(final IConsumerTransformer<IndividualResponseType, IndividualRequestType, IIndividualRequest,
+            IIndividualRequest> transformer) {
         this.transformer = transformer;
     }
 
-    // CHECKSTYLE:ON
-
-    /**
-     * @return the transformer for IndividualRequestConsumer
-     */
-    public IConsumerTransformer<IndividualResponseType, IndividualRequestType, IIndividualRequest, IIndividualRequest>
-    getTransformer() {
-        return this.transformer;
-    }

```

Comment:
Do you think it's an idea to leave this in just in case it's needed in the future?

Reply:
Maybe. As it's just a getter, I'll put it back.